doozerlib/cli/release_gen_payload.py

```diff
@@ -894,8 +897,8 @@ def get_mirroring_destination(sha256: str, dest_repo: str) -> str:
         tag = sha256.replace(":", "-")  # sha256:abcdef -> sha256-abcdef
         return f"{dest_repo}:{tag}"
 
-    @staticmethod
-    def find_payload_entries(assembly_inspector: AssemblyInspector, arch: str, dest_repo: str) -> Dict[str, PayloadEntry]:
+    @classmethod
+    def find_payload_entries(clazz, assembly_inspector: AssemblyInspector, arch: str, dest_repo: str) -> (Dict[str, PayloadEntry], List[AssemblyIssue]):
```

Comment:
i broke this method up because it was impossible to unit test. i only ended up unit testing the new stuff but more tests are welcome :)